components/Collections/src/MDCCollectionViewController.m

```diff
@@ -278,14 +278,7 @@ - (MDCInkView *)inkTouchController:(MDCInkTouchController *)inkTouchController
             inkViewAtTouchLocation:(CGPoint)location {
   NSIndexPath *indexPath = [self.collectionView indexPathForItemAtPoint:location];
   UICollectionViewCell *cell = [self.collectionView cellForItemAtIndexPath:indexPath];
-  MDCInkView *ink = nil;
-  if ([cell isKindOfClass:[MDCCollectionViewCell class]]) {
-    MDCCollectionViewCell *inkCell = (MDCCollectionViewCell *)cell;
-    if ([inkCell respondsToSelector:@selector(inkView)]) {
-      // Set cell ink.
-      ink = [cell performSelector:@selector(inkView)];
-    }
-  }
+  MDCInkView *ink = [cell performSelector:@selector(inkView)];
```

Comment:
I suspect this code was originally added because we were getting cell that were not MDCCollectionViewCells or were MDCCollectionViewCells but did not have a inkView.
There is also a chance that the cell itself will be nil.

Can we keep the original code and change the nullability annotation to nullable?
In general I prefer nonnull, but I don't think we can make that guarantee in this case.

Also, since the styler.delegate takes precedence, perhaps move that above the cell determination.